resources/plugins/MarkerOps/plugin.json

```diff
@@ -0,0 +1,13 @@
+{
+    "nameInUI": "MarkerOps (Requires MOOSE)",
+    "defaultValue": true,
+    "specificOptions": [],
+    "scriptsWorkOrders": [
+        {
+            "file": "Retribution_MarkerOps.lua",
+            "mnemonic": "MarkeOps 0.1"
```

Comment:
While this plugin doesn't have any specific options that are set through a config-file, it doesn't really break anything but I'd prefer to call thing `MooseMarkerOps` or something in that realm, basically only using alphanumeric characters and underscores 😅